### PR TITLE
vim-patch:8.1.{2355,2363}

### PR DESCRIPTION
**Pull request by janlazo**

### CODE REVIEW

**Review by blueyed — APPROVED** [review]
Note that it does not appear to be a problem for us though?
(not included with the failures there on sourcehut: https://github.com/neovim/neovim/pull/11477#discussion_r352095005)

File: src/nvim/eval.c [hunk 1]
```diff
@@ -11503,6 +11503,9 @@ static void f_glob2regpat(typval_T *argvars, typval_T *rettv, FunPtr fptr)
 static void f_has(typval_T *argvars, typval_T *rettv, FunPtr fptr)
 {
   static const char *const has_list[] = {
+#if defined(BSD) && !defined(__APPLE__)
```
Comment (janlazo) [inline src/nvim/eval.c]: Vim uses `MACOS_X`. Should `has('bsd')` work on OSX?

Comment (justinmk) [reply src/nvim/eval.c]: > Should `has('bsd')` work on OSX?

I'd say no, otherwise it makes `has('bsd')` useless...

**Comment by justinmk** [comment]
@janlazo could you add a line to `:help feature-list` ? Looks like "mac" is missing there too. (NB: that list is intentionally much smaller than Vim's)

File: runtime/doc/eval.txt [hunk 2]
```diff
@@ -4969,10 +4969,15 @@ has({feature})	Returns 1 if {feature} is supported, 0 otherwise.  The
...
 		        acl		|ACL| support
+			bsd		BSD system.
 		        iconv		Can use |iconv()| for conversion.
 		        +shellslash	Can use backslashes in filenames (Windows)
 			clipboard	|clipboard| provider is available.
+			mac		MacOS system.
+			macunix		Synonym for mac.
 			nvim		This is Nvim.
+			osx		Synonym for mac.
+			osxdarwin	Synonym for mac.
```
Comment (justinmk) [inline runtime/doc/eval.txt]: I intentionally elided variants like "osx", "win64" since they're just noise.

File: runtime/doc/eval.txt [hunk 3]
```diff
@@ -4969,10 +4969,15 @@ has({feature})	Returns 1 if {feature} is supported, 0 otherwise.  The
 <							*feature-list*
 		    List of supported pseudo-feature names:
 		        acl		|ACL| support
+			bsd		BSD system.
```
Comment (justinmk) [inline runtime/doc/eval.txt]:
```suggestion
			bsd		BSD system (not macOS, use "mac" for that).
```

**Pull request merged**